lib/email/receiver.rb

```diff
@@ -241,7 +244,7 @@ def fix_charset(mail_part)
 
     def try_to_encode(string, encoding)
       encoded = string.encode("UTF-8", encoding)
-      encoded.present? && encoded.valid_encoding? ? encoded : nil
+      !encoded.nil? && encoded.valid_encoding? ? encoded : nil
```

Comment:
`encoded.present?` can crash with "ArgumentError: invalid byte sequence in UTF-8"